### ISSUE TITLE
`Style::default_color` and `Style::on_default_color` (option to select the current default formatting)

### ISSUE THREAD

**Issue by d4h0** [jam1garner/owo-colors#6]
Hi,

After I used `owo_colors` with my changes a bit, I realized one feature was missing: It would be useful if there was a style option to apply no color style.

Let's say I want to give the user the ability to style some text, but the user doesn't want to use a color (i.e. use the default of the terminal). With the current implementation – if the default theme defines a color – the user has to choose "white" or "black". The problem is, one isn't readable on a black background, and the other isn't readable on a white background.

Basically, the following two methods are missing on `Style`:

```rust
    pub fn default_color(mut self) -> Self {
        self.fg = None;
        self
    }

    pub fn on_default_color(mut self) -> Self {
        self.bg = None;
        self
    }
```

(As always, I'm very much open to suggestions for better names)

This also adds the ability to unset colors (so "remove/unset_fg/bg` also would be possible names).

If you like this idea, let me know if I should send a PR, or if you will add it.
